src/update-process-env.js

```diff
@@ -63,7 +63,18 @@ function getEnvFromShell (env) {
     return
   }
 
-  let {stdout} = spawnSync(env.SHELL, ['-ilc', 'command env'], {encoding: 'utf8'})
+  let {stdout, error, status, signal} = spawnSync(env.SHELL, ['-ilc', 'command env'], {encoding: 'utf8', timeout: 5000})
```

Comment:
I think we should decrease the timeout, as blocking for 5 seconds will still be perceived as unresponsive. Thoughts? 💭